smac_planner/include/smac_planner/node_se2.hpp

```diff
@@ -123,6 +123,27 @@ class NodeSE2
    */
   bool operator==(const NodeSE2 & rhs)
   {
+    // TODO(james-ward) Relaxing the tolerances on the goal match
```

Comment:
Should we remove this?

Reply:
Or actually implement it? I think though with analytic expansions that this isn't really necessary.

Reply:
Agreed. I don't think we need this any more.